packages/terra-overlay/tests/jest/OverlayContainer.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import OverlayContainer from '../../src/OverlayContainer';
+import Overlay from '../../src/Overlay';
+
+describe('OverlayContainer', () => {
+  const defaultRender = <OverlayContainer />;
+  const stardardRender = (
+    <OverlayContainer>
+      <Overlay />
+      Some Text Content
+    </OverlayContainer>
+  );
+
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should have the class terra-OverlayContainer', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.prop('className')).toContain('terra-OverlayContainer');
```

Comment:
This test is more like a unit-test to check the name is present/applied. The snapshot would have the className attached, but one would have to visually spot check to ensure the name was correct. I can remove if you feel its unnecessary.